### DISCUSSION TITLE
Hide Source host via API

### DISCUSSION THREAD

**Discussion by danielsmay** [postalserver/postal#1471] [Feature suggestions]
Is it possible to hide the Source Host of a message sent via the API?

This issue here discusses it for SMTP https://github.com/atech/postal/issues/86

**Comment by danielsmay** [comment]
It should be as easy as adding a setting in the config and removing this, yes?
https://github.com/atech/postal/blob/master/app/models/outgoing_message_prototype.rb#L182

**Comment by willpower232** [comment]
Yes, arguably it could use the same setting since there probably isn't a case where you want to hide one and show the other but I'd wait for comment from atech on that front.

**Reply by ghost** [reply]
Is there any update on this? While it isn't a major issue, I'd prefer not to disclose the IP address that my service is running on in the internal network.

**Reply by willpower232** [reply]
no update, doesn't seem like anyone has implemented this feature

**Reply by amanjuman** [reply]
Are there any updates so far?
